payment-manager/src/main/java/com/qpay/paymentmanager/service/impl/PaymentServiceImpl.java

```diff
@@ -40,6 +44,14 @@ public WalletEntity makePayment(final WalletPayment walletPayment) {
         updateBalance(updatedToWalletBalance, toWallet);
 
         executorService.execute(() -> notificationClient.sendNotification(walletPayment));
+        saveTransactionToHistory(
+                fromWallet.getName(),
+                toWallet.getName(),
+                walletPayment.walletIdFrom(),
+                walletPayment.walletIdTo(),
+                walletPayment.amount()
```

Comment:
you can pass ` walletPayment` as one parameter. Then get all necessary fields in `saveTransactionToHistory` method

Reply:
i cant get names from walletPayment

Reply:
It you can get wallet ids and amount